.github/ISSUE_TEMPLATE/release.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Release Request
+about: Request a release for this package
+title: '[Release]: <version-number>'
+labels: release, high-priority
```

Comment:
This repo doesn't actually have a release tag, I'm not sure what this will do if one isn't created first.

Reply:
It does now